main.py

```diff
@@ -46,12 +46,25 @@ def read_files(args):
     # Create data instance
     data = dataset.Data()
 
-    # Read Films file
-    lines=films_file.readlines()
+    # Read files
+    films_lines=films_file.readlines()
+    collectors_lines=collectors_file.readlines()
+    collector_id=1000
+    director_id=3000
+    actor_id=2000
+    actor_set=Set()

```

Comment:
local variable yerine data classının methodları kullanılmalı.

Reply:
actor id ve director id dosyada olmadığı için bu şekilde tanımladım hocam. Dosyada tanımlı olmadığı için siz 2000 ve 3000 den başlatmamı istemiştiniz.

Reply:
Evet doğru ama yorum yaptığım satır en alttakiler :)
actor_set=Set()